rct229/rulesets/ashrae9012019/section19/section19rule4.py

```diff
@@ -0,0 +1,321 @@
+                        or ventilation_space_type_b
+                        == VENTILATION_SPACE.TRANSIENT_RESIDENTIAL_DWELLING_UNIT
+                    )
+
+                def get_calc_vals(self, context, data=None):
+                    inf_pass_cooling_b = data["inf_pass_cooling_b"]
+                    day_of_week_for_january_1 = data["day_of_week_for_january_1"]
+                    schedule_b = data["schedule_b"]
+                    space_b = context.baseline
+
+                    # check occupancy schedule
+                    multiplier_sch_occ_b = schedule_b[
+                        space_b["occupant_multiplier_schedule"]
+                    ]
+                    multiplier_sch_occ_hourly_value_b = multiplier_sch_occ_b[
+                        "hourly_values"
+                    ]
+                    design_cooling_multiplier_sch_b = multiplier_sch_occ_b[
+                        "hourly_cooling_design_day"
+                    ]
+
+                    most_used_weekday_hourly_schedule_b = (
+                        get_most_used_weekday_hourly_schedule(
+                            multiplier_sch_occ_hourly_value_b, day_of_week_for_january_1
+                        )
+                    )
+                    occ_pass_cooling_b = (
+                        most_used_weekday_hourly_schedule_b
+                        == design_cooling_multiplier_sch_b
+                    )
+
+                    # check interior lighting
+                    int_lgt_pass_cooling_b = False
+                    for interior_lighting_b in space_b["interior_lighting"]:
+                        if not int_lgt_pass_cooling_b:
+                            multiplier_sch_light_b = getattr_(
+                                interior_lighting_b,
+                                "Interior Lighting",
+                                "lighting_multiplier_schedule",
+                            )
+                            multiplier_sch_hourly_value_b = schedule_b[
+                                multiplier_sch_light_b
+                            ]["hourly_values"]
+                            design_cooling_multiplier_sch_b = schedule_b[
+                                multiplier_sch_light_b
+                            ]["hourly_cooling_design_day"]
+
+                            most_used_weekday_hourly_schedule_b = (
+                                get_most_used_weekday_hourly_schedule(
+                                    multiplier_sch_hourly_value_b,
+                                    day_of_week_for_january_1,
+                                )
+                            )
+                            int_lgt_pass_cooling_b = (
+                                design_cooling_multiplier_sch_b
+                                == most_used_weekday_hourly_schedule_b
+                            )
+
+                    # check misc equipment
+                    misc_pass_cooling_b = False
```

Comment:
Set `misc_pass_cooling_b` to True

Reply:
Addressed.

Reply:
Isn't ```False``` correct? because when ```misc_pass_cooling_b``` is set ```False``` the following loop can be implemented.